CSS/main.css

```diff
@@ -5,18 +5,33 @@
 }
 
 body {
-    background-image: linear-gradient(100deg, #575656, #062e3f);
     /* #363532, rgb(88, 111, 112) */
-    color: #ffdfdb;
     font-family: 'Work Sans', sans-serif;
     min-height: 100vh;
-    margin-top: 70px;
+    padding-top: 70px;
+}
+
+/* Body light or darker themes */
+.standard {
+    background-image: linear-gradient(100deg, #575656, #062e3f);
+    color: #ffdfdb;
+}
+
+.light {
+    background-image: linear-gradient(100deg, #EBF1F1, #B6BBBF);
+    color: #1a150e;
+}
+
+.darker {
+    background-image: linear-gradient(100deg, #211E39, #000);
+    color: white;
```

Comment:
The dark theme is also good. I guess I am not really a fan of purple/violet. XD

Here is a website - [Coolors](https://coolors.co/palettes/trending) - from where you can try really good color palettes. Let's enhance the dark mode, since we, developers, love them the best.

Reply:
Any main color you'd like to see in the darkest mode then? I'll have a look at those palettes and try to find a theme that suits better.

Reply:
Would this one look better to you?

![screenshot_darker](https://user-images.githubusercontent.com/70818055/98976835-bd8fff00-250f-11eb-9b8e-4c566aacd23e.png)

Reply:
> Any main color you'd like to see in the darkest mode then?

Any color you think that suits, leaving the decision up to you 😄 

> Would this one look better to you?

Looks good, Maybe try adding a more subtle gradient.